src/test/resources/pacts/publicapi-connector-get-payment.json

```diff
@@ -35,6 +35,8 @@
             "some_key": "key"
           },
           "amount": 100,
+          "fee": 5,
+          "net_amount": 345,
```

Comment:
i'm not sure about this. isn't net_amount = amount - fee ?

Reply:
I ran the pact locally with this and it failed telling me it expected it at 345. This is because we added corporate surcharge which gives it to 350.

Reply:
ah right, ok